features/order/managing_orders/modifying_billing_address/modifying_shipping_address_on_order_with_taxes.feature

```diff
@@ -25,6 +25,6 @@ Feature: Modifying a customer's billing address on an order with taxes
         And I specify their new billing address as "Los Angeles", "Seaside Fwy", "90802", "United States" for "Lucifer Morningstar"
         And I save my changes
         Then I should be notified that it has been successfully edited
-        And this order should be shipped to "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States"
+        And the order should be billed to "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States"
```

Comment:
I think naming of this features is wrong, cause we have both billing and shipping here. Can you fix it, please? Otherwise these changes are quite confusing.